storage/src/metadata/storage.rs

```diff
@@ -159,8 +161,8 @@ impl<B: Blob, E: Clock + Storage<B> + Metrics> Metadata<B, E> {
     }
 
     /// Get a value from `Metadata` (if it exists).
-    pub fn get(&self, key: u32) -> Option<&Bytes> {
-        self.data.get(&key)
+    pub fn get(&self, key: &K) -> Option<&Bytes> {
```

Comment:
Since this is `Bytes`, we can just do `Option<Bytes>` IMO (I know this is cleanup from my earlier work.